metrics_utility/automation_controller_billing/kubernetes_client.py

```diff
@@ -0,0 +1,51 @@
+import os
+
+from metrics_utility.exceptions import MetricsException
+from metrics_utility.logger import logger
+
+
+class KubernetesClient:
+    """
+    Simplified Kubernetes client for service account token operations.
+
+    This class assumes running in a Kubernetes pod with standard service account
+    files mounted at /var/run/secrets/kubernetes.io/serviceaccount/
+    """
+
+    def __init__(self):
+        """Initialize the client and validate service account files are available."""
+        self._validate_service_account_files()
+
+    def _validate_service_account_files(self):
+        """Validate that required service account files exist."""
+        token_path = '/var/run/secrets/kubernetes.io/serviceaccount/token'
+        namespace_path = '/var/run/secrets/kubernetes.io/serviceaccount/namespace'
```

Comment:
namespace doesn't appear to be used anywhere in the code. Shall we remove?

Reply:
yes it is not needed for the time being, I will remove